### ISSUE TITLE
Validation groups

### ISSUE THREAD

**Issue by defrag** [netmeansnet/NmnMultiUserBundle#15]
After installation it seems that it doesnt respect 'Registration' validation group. Also how can i specify valdiation groups for 
different entities?

**Comment by leopro** [comment]
fixed in master and 1.2 branches; updated documentation.

``` yaml

parameters:
  nmn_user_discriminator_parameters:
    classes:
        user_one:
            entity: Acme\UserBundle\Entity\UserOne
            registration: Acme\UserBundle\Form\Type\RegistrationUserOneFormType
            registration_options: 
                validation_groups: [Registration, Default]
            profile: Acme\UserBundle\Form\Type\ProfileUserOneFormType
            profile_options: 
                validation_groups: [Profile, Default]
            factory:
        user_two:
            entity: Acme\UserBundle\Entity\UserTwo
            registration: Acme\UserBundle\Form\Type\RegistrationUserTwoFormType
            profile: Acme\UserBundle\Form\Type\ProfileUserTwoFormType
            factory:
```

**Comment by defrag** [comment]
Works great. Thx for quick update and nice bundle :)
